lib/services/access_list.go

```diff
@@ -49,9 +49,16 @@ type AccessListsGetter interface {
 	GetAccessListsToReview(context.Context) ([]*accesslist.AccessList, error)
 }
 
+// AccessListsSuggestionsGetter defines an interface for reading access lists suggestions.
+type AccessListsSuggestionsGetter interface {
+	// GetSuggestedAccessLists returns a list of access lists that are suggested for a given request.
+	GetSuggestedAccessLists(ctx context.Context, accessRequestID string) ([]*accesslist.AccessList, error)
```

Comment:
@jakule this was not in the initial implementation, but I believe I have to add it, since `AccessListClient()` returns `services.AccessLists`.